eip/EIPS/eip-4337.md

```diff
@@ -305,6 +305,8 @@ Either `validateUserOp` or `validatePaymasterUserOp` may return a "validAfter" a
 The simulateValidation call returns this range.
 A node MAY drop a UserOperation if it expires too soon (e.g. wouldn't make it to the next block)
 
+If the `ValidationResult` includes `sigFail`, the client MUST drop the `UserOperation`.
```

Comment:
the node SHOULD drop this UserOperation

Reply:
I updated it, but why do u think it should be SHOULD instead of MUST?  You would think that there's no valid scenarios where a client should keep a UserOp with invalid sig in the mempool.